packages/jellyfish-api-core/__tests__/category/poolpair/updatePoolPair.test.ts

```diff
@@ -0,0 +1,75 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers/dist/index'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { poolpair, BigNumber } from '@defichain/jellyfish-api-core'
+import { Testing } from '@defichain/jellyfish-testing'
+import { RpcApiError } from '../../../src'
+
+describe('poolpair update', () => {
+  const container = new MasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+  const client = new ContainerAdapterClient(container)
+
+  beforeEach(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterEach(async () => {
+    await container.stop()
+  })
+
+  it('should update a pool', async () => {
```

Comment:
- `it('should update pool commission')`
- `it('should update pool ownerAddress')`
- `it('should update pool status')`
- `...`

Split up the update conditions.